index.js

```diff
@@ -92,9 +92,19 @@ function pluralTypeIndex(locale, count) {
   return pluralTypes[pluralTypeName(locale)](count);
 }
 
+function constructTokenRegex(opts) {
+  var escape = function (token) {
+    return token.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+  };
+
+  var prefix = (opts && opts.prefix) || '%{';
+  var suffix = (opts && opts.suffix) || '}';
+  return new RegExp(escape(prefix) + '(.*?)' + escape(suffix), 'g');
+}
+
 var dollarRegex = /\$/g;
 var dollarBillsYall = '$$';
-var tokenRegex = /%\{(.*?)\}/g;
+var defaultTokenRegex = constructTokenRegex();
```

Comment:
in the common case - when the prefix and suffix are the defaults - i think it'd be better to stick to the static module-cached regex literal.

Reply:
ok, makes sense, I changed it back